plasma_framework/contracts/src/exits/OutputGuardParserRegistrable.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.0;
+
+import "./IOutputGuardParser.sol";
+import "../framework/utils/Operated.sol";
+
+contract OutputGuardParserRegistrable is Operated {
```

Comment:
Minor naming question: why `Registrable`? Isn't it a `Registry`?

Reply:
At one point I thought people might prefer the pattern of `XXX is (adj)` instead of `XXX is (noun)` for mixing that why I changed my preference. But it seems like it is leading to confusion so I will change it back.